cypress-tests/cypress/support/pageBuilder/pbCreateCategoryAndBlocks.ts

```diff
@@ -1,10 +1,16 @@
 import { gqlClient } from "../utils";
-
+import { createHeadingBlock } from "./pbCreateCategoryAndBlocks/createHeadingBlock";
 interface CreateCategoryAndBlocksParams {
```

Comment:
```suggestion
import { createHeadingBlock } from "./pbCreateCategoryAndBlocks/createHeadingBlock";

interface CreateCategoryAndBlocksParams {
```

Just added an empty line, dividing the import statements and the type below them. Sorry for the nitpick 🙃